hilbert/cooccurrence/cooccurrence_sector.py

```diff
@@ -119,19 +123,24 @@ def load_coo(cooccurrence_path, include_marginals=True, verbose=True):
             add_j_idxs = add_j_idxs * sector_id.step + sector_id.j
 
             # Concatenate
+            # I is a 1D indices tensor of i(covector); J is a 1D indices tensor of j(vector)
             data = torch.cat((data, add_Nxx))
             I = torch.cat((I, add_i_idxs))
             J = torch.cat((J, add_j_idxs))
 
         if include_marginals:
             # Every sector has global marginals, so get marginals from last
             # sector.
-            Nx = torch.tensor(sector._Nx, dtype=h.utils.get_dtype())
-            Nxt = torch.tensor(sector._Nxt, dtype=h.utils.get_dtype())
+            Nx = sector._Nx.clone().detach().requires_grad_(True)
+            Nxt = sector._Nxt.clone().detach().requires_grad_(True)
```

Comment:
Lines 134, 135: why do we require grad?  We never backprop to data, so I think it should be `False`, no? 
```suggestion
            Nxt = sector._Nxt.clone().detach().requires_grad_(True)
```

Reply:
BTW, This is to get rid of the warning message right?